src/components/AddEditForm/AddEditForm.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div>
+    <form class="form-container">
+      <input-group
+        label-id="title"
+        label-text="Title"
+      >
+        <template
+          class="input-error-block"
```

Comment:
Nitpick: element with only one attribute don't need to have it in a separate line to make it more readable. 😃  
But consider looking into whether this actually works, because `template` element does not produce any element, so, this class might get lost. 🤔